OMEdit/OMEditLIB/Simulation/SimulationDialog.cpp

```diff
@@ -872,8 +872,19 @@ void SimulationDialog::initializeFields(bool isReSimulate, SimulationOptions sim
               }
               i++;
             }
-          } else { // put everything else in the Additional Simulation Flags textbox
-            additionalSimulationFlags.insert(simulationFlag, value);
+          } else { // put everything else in the Additional Simulation Flags textbox only if the simulation flag is valid
+            bool isValidSimulationFlag = false;
+            for (int i = FLAG_UNKNOWN + 1 ; i < FLAG_MAX ; i++) {
```

Comment:
If the flags were sorted alphabetically, we could do binary search here. On the other hand it's only about a hundred flags so this is probably okay until we create many more flags :sweat_smile:

Reply:
Don't think performance is an issue here. We have just around 100 flags. So is fine.